srcpkgs/python3.7-pip/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'python3.7-pip'
+pkgname=python3.7-pip
+version=18.1
+revision=1
+archs=noarch
+wrksrc="pip-${version}"
+build_style=python3-module
+pycompile_module="pip"
+hostmakedepends="python3.7-setuptools"
+depends="python3.7-setuptools"
+short_desc="PyPA recommended tool for installing PyPI packages (Python3.7)"
+maintainer="Cameron Nemo <cnemo@tutanota.com>"
+license="MIT"
+homepage="https://pip.pypa.io/"
+distfiles="${PYPI_SITE}/p/pip/pip-${version}.tar.gz"
+checksum=c0a292bd977ef590379a3f05d7b7f65135487b67470f6281289a94e015650ea1
+alternatives="
+ pip:pip:/usr/bin/pip3.7
+ pip:pip3:/usr/bin/pip3.7"
+
+post_install() {
+	test ! -e "${DESTDIR}/usr/bin/pip3" || {
```

Comment:
This should be handled by the build style, but that requires modifications that would affect python3.6 packages. Do we want to migrate to a new build style on the next python bump that uses the minor versions in the binary names?

Reply:
I just got rid of the entry points -- who really *needs* them?